cockatrice/src/localserver.cpp

```diff
@@ -11,6 +11,10 @@ LocalServer::LocalServer(QObject *parent)
 
 LocalServer::~LocalServer()
 {
+    // no need to lock clients, they live in the same thread
```

Comment:
Does the `LocalServer` class even have any locks? If not the comment is misleading. If it owns some but they are unused from lack of necessity that's even more confusing.